src/index.ts

```diff
@@ -17,6 +17,13 @@ const options = {
         default: false,
         description: 'Should imports be separated by new line ?',
     },
+    experimentalBabelParserPluginsList: {
+        type: 'path',
+        category: 'Global',
+        array: true,
+        default: [{ value: [] }],
+        description: 'Provide a list of plugins for special syntax',
+    }
```

Comment:
We can release it now. We also need to update it in the Readme.

Reply:
I updated the readme as well 👍